resources/rebuild.sh

```diff
@@ -179,16 +179,6 @@ function build_al_kernel {
         format="pe"
         target="Image"
         binary_path="arch/arm64/boot/$target"
-
-        # Patch 6.1 kernels on ARM with 6.10 patches for supporting VMGenID
-        # via DeviceTree bindings.
-        # TODO: drop this (and remove the patches from the repo) when AL backports the
```

Comment:
If the patches we have here are the same as the upstream 6.10 patches, could we just point people to the upstream and say "backport these", instead of keeping them in our repo? Because I guess otherwise we might run into danger of our copy of the patches going stale

Reply:
If our patches go stale, the commit we will be pointing to will be stale too, right?

Reply:
They won't backport it. They only back-port fixes, not new features

Reply:
No, I mean they backport something _else_ (e.g. some unrelated bugfix), and that causes our patches to no longer apply cleanly

Reply:
oh ok, got it. I _think_ it's unlikely because these patches basically change only VMGenID things, but I see what you mean (it took me a bit of time :P).

I think it makes sense, but at the same time the same would apply for the upstream commits, right?

Reply:
Yeah, but at that point, if we only say "backport these", then it's implied that the whoever follows the guide will have to work out potential conflicts themselves, instead of us updating the .patch files.

I guess long story short is I'm not a fan of keeping the patches around, although if others are fine with it, feel free to just merge with the approvals you have :)

Reply:
Nope, I think you're right. I've changed the docs to point to the PR from Linus. @Manciukic @pb8o could you please take another look to see if you're fine with that